### PR TITLE
Help and feedback links now have settable text

### PR DESCRIPTION
**Pull request by nedbat**
@jmbowman @catong

### CODE REVIEW

**Comment by catong** [comment]
Thanks @nedbat Looks good based on our conversation and my limited knowledge!

**Comment by nedbat** [comment]
@jmbowman ping.

File: edx_theme/breadcrumbs.html [hunk 1]
```diff
@@ -3,9 +3,9 @@
   <li><a href="">{{ title }}</a></li>
   <li class="wy-breadcrumbs-aside">
     {%- if help_url -%}
-      <a href="{{ help_url }}" target="_blank">Need Help?</a> |
+      <a href="{{ help_url }}" target="_blank">{{ help_link_text }}</a> or
     {%- endif -%}
-    <a href="{{ feedback_form_url }}" target="_blank">Feedback</a>
+    <a href="{{ feedback_form_url }}" target="_blank">{{ feedback_form_link_text }}</a>
```
Comment (jmbowman) [inline edx_theme/breadcrumbs.html]: Should we have a default `html_context` that gives values for these when not specified in the project using the theme?

Comment (nedbat) [inline edx_theme/breadcrumbs.html]: Or, defaults here in the template, or, only show the feedback link if there is a feedback url.

Comment (nedbat) [inline edx_theme/breadcrumbs.html]: Or, get this merged, and then do one of those! :)

**Comment by nedbat** [comment]
@jmbowman I added default text for the help and Feedback links.  
Also, can you take a look at https://github.com/edx/edx-documentation/pull/1544 ?  

Thanks.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -6,14 +6,22 @@ Change Log
...
-   
+
    This project adheres to Semantic Versioning (http://semver.org/).
 
 .. There should always be an "Unreleased" section for changes pending release.
 
 Unreleased
 ~~~~~~~~~~
 
+[1.2.0] - 2017-07-18
```
Comment (catong) [inline CHANGELOG.rst]: Date needs to be updated

**Pull request merged**